rskj.node.conf

```diff
@@ -0,0 +1,44 @@
+blockchain.config {
+    name = regtest
+    hardforkActivationHeights = {
+         bahamas = 0,
+        afterBridgeSync = -1,
+        orchid = 0,
+        orchid060 = 0,
+        wasabi100 = 0,
+        twoToThree = 0,
+        papyrus200 = 0,
+        iris300 = 0,
+        hop400 = 0,
+        hop401 = 0,
+        fingerroot500 = 0
+        arrowhead600 = 0
+    },
+    consensusRules = {
+        rskip97 = -1 # disable orchid difficulty drop
+        rskipUMM = 1
+        rskip252 = -1 # disable [transaction gas price cap](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP252.md)
+    }
+}
+peer {
+    networkId = 44444444
+}
+miner {
+    client.enabled = true
+    server.enabled = true
+    client.autoMine = false
+    # client.delayBetweenBlocks = 1 second
+    coinbase.secret = "thisisasecret"
+    minGasPrice = 1
+}
+genesis = rsk-dev.json
```

Comment:
This is necessary to override the gas limit of 6_8000_000

Reply:
I'm confused. I thought the `rskip252 = -1` did that?

Reply:
Nope, this RSKIP deals with gas price, not gas limit, you can find [here](https://ips.rootstock.io/IPs/RSKIP252.html) the spec

Reply:
We haven't got the error for the gas limit anymore. @antomor what we should do here?